src/libs/startup/outdated-browser-message.ts

```diff
@@ -0,0 +1,31 @@
+import 'outdated-browser-rework/dist/style.css';
+
+import outdatedBrowserRework from 'outdated-browser-rework';
+
+export const handleOldBrowsers = () =>
+  outdatedBrowserRework({
+    browserSupport: {
+      Chrome: 67, // Includes Chrome for mobile devices
+      Edge: 40,
+      Safari: false,
+      'Mobile Safari': 10,
+      Opera: 54,
+      Firefox: 60,
+      Vivaldi: 1,
+      IE: false,
+    },
+    isUnknownBrowserOK: true,
+    messages: {
+      en: {
+        outOfDate: 'Terra may not function correctly in this browser.',
+        update: {
+          web: `If you experience issues, please try ${(window as any).chrome ? 'updating' : 'using'} Google Chrome.`,
+          googlePlay: 'Please install Chrome from Google Play',
+          appStore: 'Please update iOS from the Settings App',
+        },
+        url: 'https://www.google.com/chrome/',
+        callToAction: `${(window as any).chrome ? 'Update' : 'Download'} Chrome now`,
+        close: 'Close',
+      },
+    },
+  });
```

Comment:
no functional change from old src/ location, but now in callable wrapper and light ts conversion